home/vim/colors/base16-tomorrow.vim

```diff
@@ -1 +1 @@
-../../../submodules/github.com/chriskempson/base16-vim/colors/base16-tomorrow.vim
\ No newline at end of file
+/home/sgreenup/code/github.com/scottgreenup/desktop/src/github.com/chriskempson/base16-vim/colors/base16-tomorrow.vim
```

Comment:
I'm a fan of using the full path, though it does mean that `sgreenup` will need to be changed for other users.
Maybe it could be an ansible task to populate the top level directories with `$HOME`?

Reply:
Crap, it is an ansible task already, it needs to be ignored.